lib/tapioca/compilers/symbol_table/symbol_generator.rb

```diff
@@ -217,6 +219,25 @@ def compile_props(constant)
           end.join("\n")
         end
 
+        sig { params(constant: Module).returns(String) }
+        def compile_enums(constant)
+          return "" unless T::Enum > constant
+
+          # T::Enum doesn't currently give a nice way to get the actual enum name so you
+          # need to parse it from the string representation which is in the form `#<SomeEnum::A>`
+          enums = T.cast(constant, T::Enum).values.map do |value|
```

Comment:
Can we rename `value` to `enum_type` to make it more obvious that we are dealing with types and not instances?
```suggestion
          enums = T.cast(constant, T::Enum).values.map do |enum_type|
```